File: models/ChatStatusType.js
const utils = require('../utils/utils');

module.exports = {
        fields: (key) => (
            {
                label: `Статус чата:  * `NEW` — новый чат. * `WAITING_FOR_CUSTOMER` — нужен ответ покупателя. * `WAITING_FOR_PARTNER` — нужен ответ магазина. * `WAITING_FOR_ARBITER` — нужен ответ арбитра. * `WAITING_FOR_MARKET` — нужен ответ Маркета. * `FINISHED` — чат завершен.  - [${key.replaceAll('__', '.')}]`,
                choices: [
                    'NEW',
                    'WAITING_FOR_CUSTOMER',
                    'WAITING_FOR_PARTNER',
                    'WAITING_FOR_ARBITER',
                    'WAITING_FOR_MARKET',
                    'FINISHED',
                ],
            }
        )
    }

File: ympa_fsharp_functions_server/src/model/ChatStatusType.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json

module ChatStatusType =

  //#region ChatStatusType

  let NEWEnum = "NEW"
  let WAITINGFORCUSTOMEREnum = "WAITING_FOR_CUSTOMER"
  let WAITINGFORPARTNEREnum = "WAITING_FOR_PARTNER"
  let WAITINGFORARBITEREnum = "WAITING_FOR_ARBITER"
  let WAITINGFORMARKETEnum = "WAITING_FOR_MARKET"
  let FINISHEDEnum = "FINISHED"
  type ChatStatusType = string

  //#endregion
